spht_project/settings.py

```diff
@@ -43,6 +45,9 @@
 
 ALLOWED_HOSTS = CONFIG.get('allowed_hosts', [])
 
+PREVIEW_MODE = True
```

Comment:
Pull this from the config so that we can change it per environment. Default should be `False`.

```python
PREVIEW_MODE = CONFIG.get('preview_mode', False)
```